### DISCUSSION TITLE
[Advanced] Adding custom css for anki cards

### DISCUSSION THREAD

**Discussion by debanjandhar12** [debanjandhar12/logseq-anki-sync#118] [Documentation]
Documentation has been moved from this location to this page: https://debanjandhar12.github.io/logseq-anki-sync/docs/advanced/custom-css

**Comment by erfanbashar1q** [comment]
hi, when I add some query to change the style of the card just the breadcrumb part will be changed; I disabled the breadcrumbs and nothing changed, is there any solution for this?

**Comment by htigraine** [comment]
Hi, I was wondering if it is possible to edit the html/javascript for the Anki front and back template and keep it permanently so without it being reverted upon syncing with Logseq? I've customised some textboxes/colours for Logseq and Anki and would love to keep it that way for card reviews.

Thank you so much for all the effort in developing this plugin! It's been a godsend for my studies, hope you have a nice week :)

**Comment by HerkulesX** [comment]
Hi, 
i was wondering if it is possible to change the Anki back template to be shown exactly like they are at Logseq. 

Like htigraine I like to highlight important informations within a Logseq block, but with the transformation in bold blue, I never have the highlights after syncing to Anki...
